_episodes/02-filedir.md

```diff
@@ -186,13 +186,13 @@ are plain old **files**.
 to use a command and what options it accepts ---
 **depending on your environment, you might find that only one of these ways works:**
 
-1. We can pass a `--help` option to the command (available on Linux and Git Bash), such as:
+1. We can pass a `--help` option to the `ls` command (available on Linux and Git Bash):
```

Comment:
This is general advice for commands, not specific to the `ls` command.

Reply:
OK, I will fix it.